server/config/config.go

```diff
@@ -176,6 +176,9 @@ func LoadConfig() (*Config, error) {
 	_ = v.BindEnv("sync.registry_monitor.check_interval")
 	v.SetDefault("sync.registry_monitor.check_interval", syncmonitor.DefaultCheckInterval)
 
+	_ = v.BindEnv("sync.auth_config.username")
+	_ = v.BindEnv("sync.auth_config.password")
+
```

Comment:
there are still leftovers from sync authconfig. is this meant to be here?

Reply:
the sync credentials are created on deploy like the zot admin credentials

Reply:
@paralta in that case, the sync config from the helm default values are required. 
we also need those inside the docker compose config